modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -81,10 +81,15 @@ echo "${forseti_environment}" > /etc/profile.d/forseti_environment.sh | sudo sh
 gsutil cp gs://${storage_bucket_name}/configs/forseti_conf_server.yaml ${forseti_server_conf_path}
 gsutil cp -r gs://${storage_bucket_name}/rules ${forseti_home}/
 
+# Download the Newest Config Validator constraints from GCS
+rm -rf $USER_HOME/policy-library
```

Comment:
How does Forseti know where to look up this directory? When we upload the Forseti rules directory we place them into `$USER_HOME/forseti-security/rules` and Forseti knows to fetch rules relative to the forseti-security repository - is there some convention that's also applied to this path?

Reply:
This folder is used by the config validator binary instead of Forseti, you can refer to the code here: https://github.com/GoogleCloudPlatform/forseti-security/blob/854004e960fea4d92697c5a3ee95bd069b8aef2c/install/gcp/scripts/initialize_forseti_services.sh#L43